tests/openthread/main.c

```diff
@@ -7,9 +7,13 @@
 #include "ot.h"
 #include "shell.h"
 #include "shell_commands.h"
-
 int main(void)
 {
+    printf("Get PANID\n");
+    uint16_t panid = 0;
+    uint8_t res = ot_call_job("panid", NULL, (void*)&panid);
```

Comment:
I would prefer to rename ot_call_job to ot_call_command. It makes more sense to me and looks more intuitive.